README.md

```diff
@@ -12,31 +12,51 @@ Since the decision to delay standardizing top-level `await` it has come up in a
 
 ## Motivation
 
-The current implementation of `async / await` only support the `await` keyword inside of `async` functions. As such we are beginning to see the following pattern emerge:
+### top-level main
```

Comment:
I think top-level main and IIAFEs are basically the same thing. Maybe fine to have as separate examples, but worth being aware.

Reply:
Indeed. I'm seeing the top-level main being used more than IIAFEs at the moment, so I was trying to illustrate that one pattern will eventually lead to the other. Perhaps that is unnecessary

Reply:
Ah OK. Might make sense to add some glue text.